chloggen/cmd/validate_test.go

```diff
@@ -48,15 +48,18 @@ func TestValidateErr(t *testing.T) {
 func TestValidate(t *testing.T) {
 	tests := []struct {
 		name    string
+		cfg     *config.Config
```

Comment:
How about changing this to something like
```suggestion
		cfgFn     func(*config.Config)
```

It looks like `config.New(t.TempDir())` is common for all test cases and only some tests needs to have the config modified.

Reply:
Fixed, thanks